### DISCUSSION TITLE
Make private methods in WebServerOrchestrator & WebClientOrchestrator internal or private protected

### DISCUSSION THREAD

**Discussion by slagtejn** [Mimetis/Dotmim.Sync#682] [Ideas]
Would you be willing to change the access modifiers on the private methods in the WebServerOrchestrator to internal?  If not maybe private protected?  If it's either of those then using 'InternalsVisibleTo' to a custom library will work.  It's hard to extend the class and use custom features without these private functions being available.  It's possible by just copying the code, but that is not a maintainable solution.

**Comment by Mimetis** [comment]
No problem, we can update the private methods to `internal virtual protected` without any problems
I've made an update to expose public methods as well.
The master branch has been updated. Let me know if it's improving your experience

**Reply by slagtejn** [reply]
Thanks!  Yes this helps a lot
